### DISCUSSION TITLE
Alternative for NextLink in mantine version 6?

### DISCUSSION THREAD

**Discussion by jt561** [mantinedev/mantine#4425] [Q&A] [answered]
Since NextLink is no longer exported from @mantine/next in v6,  what should I do? just go with a default link component from Next?

Apologies if this sounds trivial.

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/guides/polymorphic/#with-nextjs-link

**Reply by jt561** [reply]
Thank you!!
